tutorials/advanced_features/Virtual Devices.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>We usually put <code>()</code> when referring to a method (and often include the class to be unequivocal), so I would replace <code>to_virtual</code> with <code>Device.to_virtual()</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>